daemon/main.go

```diff
@@ -825,16 +825,16 @@ func runDaemon() {
 	cachesSynced := make(chan struct{})
 
 	go func() {
-		log.Info("Waiting until all pre-existing policies have been received")
+		log.Info("Waiting until all pre-existing Kubernetes resources related to policy have been received")
```

Comment:
is it related to policy or just generally Kubernetes resources?

Reply:
We don't wait for all Kubernetes resources, just the ones related to networking (Service, Ingress, NetworkPolicy, Endpoint, and CiliumNetworkPolicy) are synced. I'm open to recommendations about how to make this more clear.